tetris_description/urdf/tetris.gazebo.xacro

```diff
@@ -16,7 +16,8 @@
       <rightJoint>left_wheel_joint</rightJoint>
       <wheelSeparation>${wheel_separation}</wheelSeparation>
       <wheelDiameter>${wheel_diameter}</wheelDiameter>
-      <torque>5</torque>
+      <wheelTorque>0.8</wheelTorque>
+      <wheelAcceleration>0.05</wheelAcceleration>
```

Comment:
While we play with this value in a trial to find a good reasonable value, this will be asked.